owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/SingleSessionManager.java

```diff
@@ -111,6 +108,12 @@ public OwnCloudClient getClientFor(OwnCloudAccount account, Context context) thr
                     account.getBaseUri(),
                     context.getApplicationContext(),
                     true);    // TODO remove dependency on OwnCloudClientFactory
+
+            //the next two lines are a hack because okHttpclient is used as a singleton instead of being an
+            //injected instance that can be deleted when required
+            client.clearCookies();
```

Comment:
This behavior will change once fix/okhttp_singleton is merged however it's not really colliding with it as that PR will still provide both if these methods. So it does not relay matter here which of the changes is merged first and if these lines are still there then or not. We might however test if they can be removed once fix/okhttp_singleton is merged.